### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by oc007us** [forge/roaster#327] [General]
Greetings,

I was wondering if there is a plan to do a new release?

Thank you,
Oleg

**Comment by gastaldi** [comment]
Hey, is there any feature you're missing that's not released yet?

**Comment by oc007us** [comment]
I thought I was missing Records, but I have it. So, no worries. 

Are there any new notable features that have been added in 2.30.0-SNAPSHOT?

**Reply by gastaldi** [reply]
Nothing besides library updates so far
